### PR TITLE
Remove user id from bindings request

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

The frontend is attaching a query parameter `user_id` to every request to fetch bindings. This value isn't used in the Apps plugin, since the user's token is used instead, to identify the user. It adds noise in browser dev tools, as this parameter is always the first one occurring in the request URL

#### Ticket Link

Fixes https://mattermost.atlassian.net/browse/MM-49165

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: packages/client/src/client4.ts [hunk 1]
```diff
@@ -3687,7 +3687,6 @@ export default class Client4 {
     // to breaking changes without pushing the major version of this package.
     getAppsBindings = async (userID: string, channelID: string, teamID: string) => {
```
Comment (hanzei) [inline packages/client/src/client4.ts:3688]: Should `userID` also get removed here?

**Review by hanzei — CHANGES_REQUESTED** [review]

**Review by hanzei — APPROVED** [review]

**Review by julmondragon — CHANGES_REQUESTED** [review]
Changes looks good @mickmister !

However some tests needs to be updated: https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/64872/workflows/f09204d3-0516-4e63-82a5-1a6ca2fa5dd7/jobs/440642

And there is a lint issue: https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/64872/workflows/f09204d3-0516-4e63-82a5-1a6ca2fa5dd7/jobs/440641

**Review by julmondragon — APPROVED** [review]
Thank you @mickmister !

**Pull request merged**